hack/central-repo/upload-plugins.sh

```diff
@@ -62,6 +62,15 @@ CREATE TABLE IF NOT EXISTS "PluginBinaries" (
 		"URI"                TEXT NOT NULL,
 		PRIMARY KEY("PluginName", "Target", "Version", "OS", "Architecture")
 	);
+
+CREATE TABLE IF NOT EXISTS "PluginGroups" (
+		"Publisher"          TEXT NOT NULL,
```

Comment:
Using `Publisher` here to specify `vmware-tkg` seems confusing. As we are using `Publisher` in the `PluginBinaries` table for different meanings where the value will only be `tkg`.

So, I would either rename the column name to `VendorPublisher` or make it consistent and add the `Vendor` and `Publisher` columns separately. My preference will be to have 2 separate columns.

Reply:
After discussion we agree to have an explicit `Vendor` and `Publisher`.
When an individual eventually publishes a plugin, we could have `Vendor = "independent"`, or even keep the `Vendor` nil.